deploy.sh

```diff
@@ -40,34 +41,40 @@ npm install
 grunt build
 
 # Stop Mes Aides
-killall --user `whoami` node || echo 'No server was running'
+forever stop mes-aides || echo 'No server was running'
 
-mongo localhost --eval "db.copyDatabase('mes-aides-master', '`whoami`')"
+mongo localhost --eval "db.copyDatabase('mes-aides-master', '$USER')"
 
 # Start Mes Aides
-OPENFISCA_URL="http://localhost:$OPENFISCA_PORT" SESSION_SECRET=foobar NODE_ENV=production MES_AIDES_ROOT_URL="$PROTOCOL://$PUBLIC_HOST" PORT=$PORT MONGODB_URL="mongodb://localhost/$(whoami)" nohup node server.js >> ../mes-aides.log 2>> ../mes-aides_error.log &
+OPENFISCA_URL="http://localhost:$OPENFISCA_PORT" SESSION_SECRET=foobar NODE_ENV=production MES_AIDES_ROOT_URL="$PROTOCOL://$PUBLIC_HOST" PORT=$PORT MONGODB_URL="mongodb://localhost/$USER" forever --uid mes-aides -l ../mes-aides.log -e ../mes-aides_error.log --append start server.js
 
 cd ..
 
 # Install OpenFisca
 if ! cd openfisca
 then
-    git clone https://github.com/sgmap/openfisca.git --branch $TARGET_BRANCH
```

Comment:
To be robust in case there is no `openfisca` target branch (which happens a lot in real life, when we are just trying to deploy something new on the front). Getting the master version is ok here, as this is just to get the `update.sh file`. The rest of the code will be updated by `update.sh` itself.